docs/markdown/Include-directories.md

```diff
@@ -27,4 +27,6 @@ proper compiler flags to make it all work.
 
 Another thing to note is that `include_directories` adds both the
 source directory and corresponding build directory to include path, so
-you don't have to care.
+you don't have to care. This can be disabled with the
+`implicit_include_directories` argument to the [build
+function](Reference-manual_functions.md) you use.
```

Comment:
Possibly:

> Another thing to note is that the default include directories adds both [...] to include path, since most people seem to want it. If it turns out you don't want it after all, this can be disabled with [...]

Reply:
Thanks, I've amended the PR with your version